src/components/new-safe/create/steps/StatusStep/index.tsx

```diff
@@ -53,7 +53,6 @@ export const CreateSafeStatus = ({
     if (!chain || !safeAddress) return
 
     if (status === SafeCreationEvent.SUCCESS) {
-      dispatch(updateAddressBook(chain.chainId, safeAddress, data.name, data.owners, data.threshold))
```

Comment:
Why are we no longer updating the addressBook?

Reply:
We are still updating the address book but earlier when you submit the tx / create the Safes.

Reply:
In `ReviewStep.tsx`